src/App.js

```diff
@@ -1,18 +1,17 @@
 import React from 'react'
 import Dashboard from './components/Dashboard'
 import Login from './components/Login'
-import Navigation from './components/Navigation'
+import Register from './components/Register'
 import {Routes, Route} from 'react-router-dom'
 import './styles/main.scss'
 
 function App() {
   return (
     <div>
-      <Navigation/>
       <Routes>
-        <Route path='/'/>
+        <Route path='/' element={<Dashboard/>} />
```

Comment:
The /dashboard should take you to Dashboard. Use redirect for that.
https://v5.reactrouter.com/web/api/Redirect